src/compute-client/src/controller/instance.rs

```diff
@@ -1698,21 +1699,21 @@ where
         replica_id: ReplicaId,
     ) {
         if !self.collections.contains_key(&id) {
-            soft_panic_or_log!(
+            warn!(
```

Comment:
Oh, forgot to review these!

This was necessary because now we can drop replicas while they still have messages in the response channel? In that case I think this should be a silent ignore, rather than a warning.

But another thing worries me: If a replica fails and is reconnected, but the old incarnation still has responses in the channel, won't we get confused when we read these responses later? When we reconnect a replica, we re-initialize all its state as if the replica was dropped and added again, so I can imagine seeing responses from the old replica would be unexpected.

Seems like we need to also tag responses in the channel with the replica epoch, so we can ignore them if the epoch doesn't match that of the current replica.